openassessment/templates/openassessmentblock/message/oa_message_complete.html

```diff
@@ -1,13 +1,13 @@
 {% load i18n %}
 {% spaceless %}
 <div id="openassessment__message__{{ xblock_id }}" class="openassessment__message message">
-    <h3 class="message__title">{% trans "You Have Completed This Assignment" %} </h3>
+    <h5 class="message__title">{% trans "Status" %} </h5>
     <div class="message__content">
         <p>
             {% if waiting %}
-                <p>{% trans "Your final grade will be available when the assessments of your response are complete." %}</p>
+                <p>{% trans "You Have Completed This Assignment. Your final grade will be available when the assessments of your response are complete." %}</p>
             {% else %}
-                <a data-behavior="ui-scroll" href="#openassessment__grade__{{ xblock_id }}">{% trans "Review your grade and your assessment details." %}</a>
+                <a data-behavior="ui-scroll" href="#openassessment__grade__{{ xblock_id }}">{% trans "You Have Completed This Assignment. Review your grade and your assessment details." %}</a>
```

Comment:
The titles that have been moved into the message body should be sentence case: "You have completed this assignment."